src/solver/variable/commons/join.h

```diff
@@ -210,42 +221,50 @@ class Join : public Variable::IVariable<Join<LeftT, RightT>, Yuni::Default, VCar
     {
         LeftType::yearEndBuild(state, year, numSpace);
         RightType::yearEndBuild(state, year);
+        BindConstType::yearEndBuild(state, year, numSpace); // gp : need this call ?
     }
 
     void yearEnd(unsigned int year, unsigned int numSpace)
     {
         LeftType::yearEnd(year, numSpace);
         RightType::yearEnd(year, numSpace);
+        BindConstType::yearEnd(year, numSpace); // gp : need this call ?
     }
 
     void computeSummary(std::map<unsigned int, unsigned int>& numSpaceToYear,
                         unsigned int nbYearsForCurrentSummary)
     {
         LeftType::computeSummary(numSpaceToYear, nbYearsForCurrentSummary);
+        BindConstType::computeSummary(numSpaceToYear, nbYearsForCurrentSummary);
```

Comment:
**Caution** : there is an error here : function **BindConstType::computeSummary** is called twice.
This error is fixed in a further PR (which is : \<PR\>)